inventory/splunk_defaults_linux.yml

```diff
@@ -83,6 +83,8 @@ splunk:
     multisite_replication_factor_total: 3
     multisite_search_factor_origin: 1
     multisite_search_factor_total: 2
+    license_master_url:
+    cluster_master_url:
```

Comment:
Semi-related: we have a `splunk.shc.enable` and `splunk.idxc.enable` which at one point in time may have been used to turn on clustering. But it seems like that is no longer the case and nothing uses those two `enable` flags. Should we consider removing them?

Reply:
I confirmed that the enable flags were not being used anywhere, and removed references to them in the latest commit. I also moved the calculated booleans out of environ.py and into the plays themselves, which I think is less confusing.